assets/src/tinymce/__tests__/components/controls/Toggle.test.js

```diff
@@ -0,0 +1,55 @@
+/*
```

Comment:
Why are these files in a directory called `__tests__`. Why not tests?

Reply:
@spacedmonkey  We have this in our configuration `jest.config.js`. If we explicitly want all tests to reside in `tests` directory, we can remove other two rules (1st and 3rd) from config and modify 2nd rule as `'**/tests/**/*.[jt]s',`

```js
  testMatch: [
    '**/__tests__/**/*.[jt]s',
    '**/test/**/*.[jt]s',
    '**/?(*.)test.[jt]s',
  ]
```

Reply:
While `__tests__` is supported, it's so far uncommon in our code base. Typically we've added tests in a `test` folder closer to the files/components they're testing.

But it's fine. We can leave it like this for now. We can always shuffle files around later if we  want to.